crates/sequencer/Cargo.toml

```diff
@@ -13,8 +13,12 @@ license.workspace = true
 
 [dependencies]
 # Starknet
-blockifier = { git = "https://github.com/starkware-libs/blockifier.git", tag = "v0.3.0-rc0" }
+blockifier = { package = "blockifier", git = "https://github.com/starkware-libs/blockifier.git", tag = "v0.3.0-rc0" }
```

Comment:
Currently this is being patched by the blockifier patch we have in the workspace manifest. Once we remove the Katana dep, we can remove the patch and this dependency should be applied.

Reply:
Can we add a todo?